theano/compile/profiling.py

```diff
@@ -1391,6 +1391,18 @@ def exp_float32_op(op):
                           " generator supported on the GPU.", file = file)
                 break
 
+        # tip 6
+        for a in self.apply_time:
+            node = a
+            print("dtype", [i.dtype for i in node.inputs])
+            print("dim", [i.ndim for i in node.inputsg])
```

Comment:
inputsg -> input

This is what make travis fail. In fact, remove those 2 print. We dont, want them printed.